create_jenkins_job.py

```diff
@@ -216,9 +216,9 @@ def create_job(os_name, job_name, template_file, additional_dict):
             packaging_label_expression = 'macos &amp;&amp; mojave'
 
         # configure a manual version of the packaging job
-        ignore_rmw_default_packaging = {'rmw_opensplice_cpp'}
```

Comment:
As we discussed privately, we shouldn't remove this yet.  Once dashing goes out of support, we can remove all of the Opensplice support together.

Reply:
Fixed in #552